docker/docker-run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#plagiarized from wangxian/alpine-mysql
```

Comment:
Is it compatible with their licensing to redistribute the install script like this?